README.md

```diff
@@ -607,6 +609,8 @@ These can be useful for osint and social engineering.
 - [FamilySearch](https://www.familysearch.org/search/) - Search for your ancestors in birth certificates, marriage registrations, census records, and other documents
 - [iTools - Wink People Search](http://itools.com/tool/wink-people-search) - Free people search. Find people on social networks and across the Web
 - [Intelius](https://www.intelius.com/) - Leading provider of public data about people and their connections to others
+- [Whatsmyname](https://whatsmyname.app/) - OSINT Combine username search tool
```

Comment:
Description should be `OSINT tool designed for enumerating usernames across a wide array of websites`